### DISCUSSION TITLE
採購專業人員基礎證照測驗練習

### DISCUSSION THREAD

**Discussion by YiChiFang** [sdwh/sdwh.github.io#32] [Q&A]
版主您好：
採購專業人員基礎證照測驗練習
題目請教要如何更新呢？或是版主會協助更新呢？
感謝您！

**Comment by ilovesu** [comment]
版主您好，感謝您提供採購測驗網站，但題目做到一半，畫面全白了。請求協助。謝謝您。
https://5tw.github.io/gpa/#/

**Reply by sdwh** [reply]
@ilovesu 哈囉，您是否可以使用 Chrome 或者 Edge，使用 F12 提供截圖 Console 的相關訊息呢？
另這個測驗資料都是存在您電腦的瀏覽器上，要請您提供 Application Local Storage 中的答題情形資料，以利判斷。

![](https://i.imgur.com/8bBcyPY.png)

此外最直接的復原方式就是刪除掉 Application 中 Local Storage 的資料，但答題記錄就會消失了 😥

**Comment by sdwh** [comment]
@YiChiFang 哈囉，我會不定期確認採購法題庫的情況，並進行更新唷～

**Comment by ilovesu** [comment]
謝謝您，功德無量

可否跟您加Line好友
我用SAFARI就不會出現空白了

我的 Line ID
weloveteachersu


sdwh ***@***.***> 於 2022年7月20日 週三 下午6:13寫道：

**Reply by sdwh** [reply]
哈囉 @ilovesu ：

要謝謝您的回饋，測過一番測試發現網站原本確實存在問題，會在特定的情況下造成全白畫面的問題，已經完成修正。

並且配合採購法題庫更新，更新系統囉！如果是桌面瀏覽器，可以使用 Ctrl + F5 進行重整，以取得最新的網站更新😀

**Comment by ilovesu** [comment]
所以，可以用chrome了是嗎

sdwh ***@***.***> 於 2022年7月23日 週六 下午1:21寫道：
